contacts-overview.adoc

```diff
@@ -39,4 +39,4 @@ Right-click any listed contact to see further options like **Edit**, **Delete**,
 
 When a contact is selected, this pane displays its details.
 
-This pane offers options like **Edit Details**, **Delete**, *Print*, and **Assign to Lists**.
\ No newline at end of file
+This pane offers options like **Edit Details**, **Assign to Lists**, **Move**, and **Delete**.
```

Comment:
Will it be a good idea to mention about 3-dot menu options Print and Tag?

Reply:
Fixed